apps/hash-graph/lib/graph/src/store/postgres/knowledge/entity.rs

```diff
@@ -221,6 +221,28 @@ impl<C: AsClient> PostgresStore<C> {
 
         Ok(())
     }
+
+    #[tracing::instrument(level = "trace", skip(self))]
+    #[cfg(hash_graph_test_environment)]
+    pub async fn delete_entities(&mut self) -> Result<(), DeletionError> {
+        self.as_client()
+            .client()
+            .simple_query(
+                r"
+                    DELETE FROM entity_has_left_entity;
+                    DELETE FROM entity_has_right_entity;
+                    DELETE FROM entity_is_of_type;
+                    DELETE FROM entity_temporal_metadata;
+                    DELETE FROM entity_editions;
+                    DELETE FROM entity_ids;
```

Comment:
Yes, we would need to add `ON DELETE CASCADE` to the database, which I tried to avoid. Being explicit here isn't actually that bad as we only can remove everything, not only specific entries.